bugbear.py

```diff
@@ -1192,8 +1192,8 @@ def visit_Lambda(self, node):
 }
 B015 = Error(
     message=(
-        "B015 Pointless comparison. This comparison does nothing but waste "
-        "CPU instructions. Either prepend `assert` or remove it."
```

Comment:
I personally feel the blunt nature of the message makes it clearer if what the check looks for.
The new message will make people think into it more and possibly over complicate their thoughts. But open to others thoughts …

Reply:
I definitely see the tradeoff here, and agree that the cause of the error should be the first thing the user sees. The aim with the change of wording was to retain that, while softening the order to a suggestion.